src/main/java/Parser.java

```diff
@@ -0,0 +1,62 @@
+public class Parser {
+
+    public Parser() {
+    }
+
+    public static void parse(String command, TaskList tasks, Storage storage) {
+        if (command.equals("list")) {
```

Comment:
Perhaps instead of multiple if else block, you can use switch instead? It would make the code more readable